game-core/src/main/java/games/strategy/engine/framework/startup/ui/editors/SelectAndViewEditor.java

```diff
@@ -103,7 +101,7 @@ private void updateView() {
     }
     view.removeAll();
     final IBean item = (IBean) selector.getSelectedItem();
-    editor = item.getEditor();
+    editor = item == null ? null : item.getEditor();
```

Comment:
Nit: I was going to request that you surround `item == null` with parentheses so I don't go cross-eyed due to the preceding assignment operator. :smile:  But an

```java
Optional.ofNullable((IBean) selector.getSelectedItem()).map(IBean::getEditor).orElse(null)
```
would work too, if you prefer that kind of thing.

Reply:
Not really a fan of uneccessary parentheses at (personal opinion) obvious places.
I thought about using an optional but I dropped this idea because I thought that the line would get too long.
However I only considered this for this exact line, I didn't consider merging the preceding line in, so using an optional seems like a good choice.